frontend/js/src/missing-mb-data/MissingMBData.tsx

```diff
@@ -185,27 +201,63 @@ export default class MissingMBDataPage extends React.Component<
                 <Loader isLoading={loading} />
               </div>
               {missingData.map((data, index) => {
-                const additionalActions = (
-                  <ListenControl
-                    buttonClassName="btn btn-sm btn-success"
-                    icon={faPlus}
-                    title="Add missing recording"
-                    text=""
-                    // eslint-disable-next-line react/jsx-no-bind
-                    action={this.submitMissingData.bind(
-                      this,
-                      missingMBDataAsListen[index]
-                    )}
-                  />
-                );
+                let additionalActions;
+                if (currentUser?.auth_token) {
+                  const addToMB = (
+                    <ListenControl
+                      buttonClassName="btn btn-sm btn-success"
```

Comment:
I think we should encourage users to use the "link with…" button first rather than the add button, which we generally fear could lead to creating a lot of duplicates and bad data.
As such, maybe the `btn-success` class should instead go to the other button below?

Reply:
And for posterity, I'll add that in order to make the link-to-mb modal actually user friendly (and especially relevant on this page perhaps), we'll want to allow users to search for MB recordings rather than only expecting pasted MBIDs/MB URLs.
I guess we can revisit that on the back of #2331 which tackles some of the same questions about searching for a recording.